Cesium3DTilesSelection/src/Tile.cpp

```diff
@@ -307,42 +308,38 @@ void* processNewTileContent(
 } // namespace
 
 void Tile::loadContent() {
-  if (this->getState() != LoadState::Unloaded) {
-    // No need to load geometry, but give previously-throttled
-    // raster overlay tiles a chance to load.
-    for (RasterMappedTo3DTile& mapped : this->getMappedRasterTiles()) {
-      mapped.loadThrottled();
-    }
-    return;
-  }
+  assert(this->getState() == LoadState::Unloaded);
 
   this->setState(LoadState::ContentLoading);
 
   Tileset& tileset = *this->getTileset();
 
-  std::optional<Future<std::shared_ptr<IAssetRequest>>> maybeRequestFuture =
-      tileset.requestTileContent(*this);
-
-  if (!maybeRequestFuture) {
-    // There is no content to load. But we may need to upsample.
-
-    const UpsampledQuadtreeNode* pSubdivided =
-        std::get_if<UpsampledQuadtreeNode>(&this->getTileID());
-    if (pSubdivided) {
-      // We can't upsample this tile until its parent tile is done loading.
-      if (this->getParent() &&
-          this->getParent()->getState() == LoadState::Done) {
+  // If this is an upsampled tile, we need to derive this tile's content from
+  // its parent.
+  const UpsampledQuadtreeNode* pSubdivided =
+      std::get_if<UpsampledQuadtreeNode>(&this->getTileID());
+  if (pSubdivided) {
+    // We can't upsample this tile until its parent tile is done loading.
+    if (this->getParent()) {
+      if (this->getParent()->getState() == LoadState::Done) {
         std::vector<Projection> projections = mapOverlaysToTile(*this);
         this->upsampleParent(std::move(projections));
-      } else {
+      } else if (this->getParent()->getState() != LoadState::Unloaded) {
```

Comment:
Hm that's right, I'm trying to think why we thought that was necessary yesterday. It's the case now that a tile _must_ have loadable content for `loadContent` to be called, but it turns out it only makes sense to ever call `continueLoadingContent` when a tile has loadable content as well. Otherwise, there will be no model and therefore no need to update rasters that don't exist.